src/Interval.cpp

```diff
@@ -98,5 +98,8 @@ Interval Interval::make_intersection(const Interval &a, const Interval &b) {
 Expr Interval::pos_inf = Variable::make(Handle(), "pos_inf");
 Expr Interval::neg_inf = Variable::make(Handle(), "neg_inf");
 
+/*static*/ Expr Interval::get_pos_inf() { return pos_inf; }
```

Comment:
I really don't like the commented out static as it is ambiguous as to whether it is an annotation or commented out syntax. If you want to add a useful comment like ```/* Static getter methods to avoid exporting constants. */``` above the two functions, that seems more explanatory.

Reply:
Interesting; I always like it because I feel that it makes the result *more* readable (since the staticness is only reflected syntactically in the .h file); the fact that recapitulation of it here isn't required by C++ strikes me as a language defect. But I can live with a more verbose way of doing the same thing.